packages/reports/addon/components/asset-owner.hbs

```diff
@@ -0,0 +1,2 @@
+{{!-- Copyright 2022, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{@user}} - {{this.updatedon}}
```

Comment:
Can we add something like `<DenaliIcon @icon="user-single" @size="extrasmall" />` to the front